### PR TITLE
Join request from feature branch to develop branch

### PR DESCRIPTION
**Pull request by aristides1000**
I request that the possibility of joining the feature branch in the develop branch be studied.
This web page was the second project of the Ruby on Rails programming language of the Microverse curriculum.
In this project, I learned the importance of forms applications and MVCs in developing real-world solutions.

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED 🔴 ♻️ 

## Hey Happy Coder, 👋🏽 
**This is a great project you have going on, well-done lad**. 

### Well-done 💯 
    1. The app runs just fine. You got the validations right. 👍🏽 

**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

### REQUIRED

- [x] Add clear instructions on how to get started with your app on a remote computer. Remember to include `bundle install`, `yarn`, `rails db:migrate`, `rails s`
- [x] Clean up the readme file. A few sections include information about web scraping.
- [x] In the new file of the views folder, You need the commented-out code of the `form` and the `form_tag` and then keep the `form_with`. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form1.png) [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form2.png) [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form_tag2.png) from Odin)
- [x] For the edit file in the views folder, You are required to use the `form_for` tag. [Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form_edit.png)
- [x] Check inline comments for more sections to improve

#

### Kindly make all the requested changes and submit another CRR. 💯 

**Cheers 🍹 and Happy coding 💻** 

<hr>

Mention @mke2111 in this thread and I'll be glad to help. 😄 
### Best regards
Slack 🆘  Roy Mukuye
Twitter 🐦  [Roy Mukuye](https://twitter.com/Roymkenya)
LinkedIn 🔗  [Roy Mukuye](https://www.linkedin.com/in/roy-mukuye/)

File: config/routes.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Rails.application.routes.draw do
+  root 'users#index', as: '/'
+  resources :users, only: [:index, :new, :create, :update, :destroy, :show, :edit]
```
Comment (mke2111) [inline config/routes.rb]: - [ ] Use only these actions. `[:new, :create, :edit, :update]`

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    @user = User.new(user_params)
+
+    if @user.save
+      redirect_to new_user_path
+    else
+      render :new
+    end
+  end
+
+  def edit; end
```
Comment (mke2111) [inline app/controllers/users_controller.rb]: - [x] The edit action finds user based on id from params, i.e. `@user = User.find(params[:id])` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_route2.png) from Odin)

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# STATUS: INVALID PULL REQUEST :stop_sign: 

## Hello @aristides1000, I think you've made a mistake while submitting this project. You submitted this for the Authentication project while this project is for the Forms project. Kindly ask for another code review.

# HAPPY CODING :computer: :desktop_computer:

**Review by GzimAsani — APPROVED** [review]
# STATUS: Project Accepted :green_circle: :partying_face: 

## Hello again @aristides1000, thank you for making all the requested changes. Your project is approved and good luck with your next projects :+1: 

# HAPPY CODING :computer: :desktop_computer:

**Pull request merged**